internal/rundata/dut.go

```diff
@@ -27,22 +28,39 @@ import (
 	"github.com/openconfig/ondatra/gnmi/oc"
 	"github.com/openconfig/ondatra/gnmi/oc/ocpath"
 	"github.com/openconfig/ygnmi/ygnmi"
+
+	gpb "github.com/openconfig/gnmi/proto/gnmi"
 )
 
-// dutInfo retrieves the vendor, model, and OS version from the device from various
+// DUTInfo retrieves the vendor, model, and OS version from the device from various
 // OpenConfig paths.
-type dutInfo struct {
+type DUTInfo struct {
 	vendor string
 	model  string
 	osver  string
```

Comment:
Let's just export the fields so we don't need to add the accessor functions.

Also, Go initialism would export `osver` as `OSVer`.

Reply:
Done!